config/config-example.js

```diff
@@ -608,6 +608,15 @@ exports.grouplist = [
 		alts: 's',
 		broadcast: true,
 	},
+	{
+		symbol: '\u200B',
+		id: "whitelist",
+		name: "Whitelist",
+		inherit: ' ',
+		alts: 's',
+		roomonly: true,
+		showimage: true,
```

Comment:
This seems to have found itself back here. :|

If it's to be here at all, it needs to be accurate to the new implementation...

```suggestion
		broadcast: true,
		broadcastimage: true,
	},
	{
		symbol: 'whitelist',
		id: "whitelist",
		name: "Whitelist",
		inherit: ' ',
		roomonly: true,
		alts: 's',
		broadcast: true,
		broadcastimage: true,
```